{{cookiecutter.project_name}}/docker/django/Dockerfile

```diff
@@ -19,19 +19,18 @@ ENV DJANGO_ENV=${DJANGO_ENV} \
 
 # System deps:
 
-RUN apk update \
-    && apk --no-cache add \
-      bash \
-      build-base \
-      curl \
-      gcc \
-      gettext \
-      git \
-      libffi-dev \
-      linux-headers \
-      musl-dev \
-      postgresql-dev \
-      tini
+RUN apk --no-cache add \
```

Comment:
These changes were already included, please remove them.

Reply:
i don't understand  what to remove?

Reply:
These changes should not be presented here.
You need to undo these changes.

Since it have been already merged in #398.

The easiest way to do it:
0. Save backups of `Pipfile` and `Pipfile.lock` that you have edited
1. Delete your fork and your local folder
2. Fork this repo again
3. Paste modified `Pipfile` and `Pipfile.lock` back again
4. Create new pull-request

But if you want to do it the-right-way:
1. See how branching works
2. See how `rebase` works
This is time consuming, it may take you a lot of time to get comfortable with.